spec/lib/spree_sitemap/spree_defaults_spec.rb

```diff
@@ -27,6 +27,18 @@
     end
   end
 
+  context ".gem_available?" do
+    it "verifies that gem is available" do
+      expect(subject.gem_available?('rspec-rails')).to be_truthy
+    end
+
+    context "when there is no such gem" do
+      it "returns false" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.